### ISSUE TITLE
Provide security.txt for all FlowForge installs

### ISSUE THREAD

**Issue by ZJvandeWeg** [flowforge/flowforge#868]
### Description

A security.txt file is somewhat standard to have and fairly simple to implement. It tells security researchers where and how to file reports.

See also: https://securitytxt.org/

**Comment by hardillb** [comment]
I thought we already had an open issue for this (or at least one for the `flowforge.com` domain.

But we should be able to set up a wild card route to serve one for all project domains on `flowforge.cloud`

**Comment by ZJvandeWeg** [comment]
Can't we include it into self managed installs too?Op 10 aug. 2022 12:10 schreef Ben Hardill ***@***.***>:
I thought we already had an open issue for this (or at least one for the flowforge.com domain.
But we should be able to set up a wild card route to serve one for all project domains on flowforge.cloud

**Comment by hardillb** [comment]
Having k8s serve up a file for all project host names is pretty simple (a single wildcard currently then one per custom domain). Docker will be harder, but should be possible by injecting some nginx config.

But would require collecting information from the admin to populate it properly (and possibly a gpg key to sign it...)

I don't think it makes sense for localfs.

**Comment by knolleary** [comment]
Need to clarify the requirement. No one said anything about individual hosted projects serving up that file. I strongly think they shouldn't.

But the core platform itself could easily do so... If that's the req.

**Comment by sammachin** [comment]
I agree with Nick that projects shouldn't provide a default security.txt, we don't want to receive reports of vulnerabilities in peoples flows those should goto the owner.

(Would be a good article to write about adding and serving up a securty.txt from node-red)

In terms of self managed instances, we should have a generic/sample security.txt as part of the install and then the admin should be encouraged to edit it as appropriate during deployment. The same file should be functional but make it clear that it is not a flowforge managed instance.
We can then make the relevant edits for app.flowforge.cloud
